tests/test_loaders.py

```diff
@@ -243,3 +243,14 @@ def test_csv_pandas_loader_no_encoding(self, tmp_path, noaa_jfk_schema):
 
         del noaa_jfk_schema['subdatasets']['jfk_weather_cleaned']['format']['options']['encoding']
         self.test_csv_pandas_loader(tmp_path, noaa_jfk_schema)
+
+    def test_csv_pandas_header(self, tmp_path, noaa_jfk_schema):
+        "Test CSVPandasLoader header options"
+
+        noaa_jfk_schema['subdatasets']['jfk_weather_cleaned']['format']['options']['header'] = True
+        self.test_csv_pandas_loader(tmp_path, noaa_jfk_schema)
+
+        noaa_jfk_schema['subdatasets']['jfk_weather_cleaned']['format']['options']['header'] = False
+        with pytest.raises(ValueError) as exinfo:  # Pandas should error from trying to read string as another dtype
+            Dataset(noaa_jfk_schema, tmp_path, mode=Dataset.InitializationMode.DOWNLOAD_AND_LOAD)
+            assert('could not convert string to float' in exinfo.value)
```

Comment:
The exception is raised in the previous line and this assertion should have never been executed:

```suggestion
        assert 'could not convert string to float' in str(exinfo.value)
```

Reply:
Whoops thanks for catching